src/npm/types.ts

```diff
@@ -45,6 +45,7 @@ export interface GetVersion {
   schematics?: string;
   types?: string;
   typings?: string;
+  style?: string;
```

Comment:
The type is currently in bad state to comply with old code during migration (I should add a warning). 
Style/typings and such are not returned in the root level but in the `versions` key.
See Bootstrap for example: https://registry.npmjs.com/bootstrap

NicePackage support merging root level / versions key, but it's incorrect; this will change a bit your tests but not the logic.

Reply:
I'm not sure I understood you correctly so let me clarify this - it's fine to have the `style` field in the GetVersion interface. But I violate that interface in my tests and that's not good 🙂 

I must change the tests to follow the interface:
```typescript
expect(
  formatPkg({
    ...BASE,
    versions: { // this field with its subfields was missing in original tests
      '1.0.0': { ...BASE_VERSION, style: '/style.min.css' }
    },
  })?.styleTypes
).toEqual(['css']);
```

Reply:
yes exactly (sorry I was not really clear)
style only exists in the GetVersion interface

Reply:
https://github.com/algolia/npm-search/pull/683/commits/4be334b8d4fd2ae7ddfb3728b9bdcf7a5cd7dc90